interface/billing/search_payments.php

```diff
@@ -42,7 +47,7 @@
 
     //delete and log that action
         row_delete("ar_session", "session_id ='" . add_escape_custom($DeletePaymentId) . "'");
-        row_delete("ar_activity", "session_id ='" . add_escape_custom($DeletePaymentId) . "'");
+        row_modify("ar_activity", "deleted = NOW()", "deleted IS NULL AND session_id ='$DeletePaymentId'");
```

Comment:
is there a reason that not using the add_escape_custom() here to prevent sql-injection?

Reply:
Thanks, I just missed it.